etl/dw/f_geracao_distribuida.py

```
@@ -0,0 +1,228 @@
+        i = 0
+        for i, c in enumerate(self.cursor, 1):
+            _list.extend([c])
+            if i % self.chunksize == 0:
+                yield _pd.DataFrame(_list)
+                _list = []
+
+        yield _pd.DataFrame(_list)
+
+    @_log
+    def treat(self):
+        df_extract = self.df_extract.assign(
```

Ruff (0.12.2)

Local variable `df_extract` is assigned to but never used

Remove assignment to unused variable `df_extract`

(F841)